package.json

```diff
@@ -45,6 +47,7 @@
     "grunt-qunit-istanbul": "0.4.5",
     "grunt-search": "0.1.6",
     "load-grunt-tasks": "0.3.0",
+    "qunit-reporter-stdout": "0.1.0",
```

Comment:
If this is only a devDependency, how does `QUnit.stdout()` actually add any value? Is all we're removing from userland a single require-call?